uwp/DataGrid/Editing.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Editing in UWP DataGrid control | Syncfusion
-description: Learn here all about Editing support in Syncfusion UWP DataGrid (SfDataGrid) control and more.
+title: Editing in UWP DataGrid Control | Syncfusion
+description: Learn here all about to enable and disable editing support in Syncfusion UWP DataGrid control, its elements, and more.
```

Comment:
Update like this - Learn here all about editing support in Syncfusion UWP DataGrid control, its elements, and more.

Reply:
Comment resolved @gowthamanthavasiyappan